File: Selenium/Project/Activity5.java
package projects;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity5 {
    public static void main(String[] args){
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();

        driver.get("https://alchemy.hguy.co/lms");

        driver.findElement(By.xpath("//a[text()='My Account']")).click();

        String title = driver.getTitle();

        if(title.equals("My Account – Alchemy LMS")){
            System.out.println("The title of the page is My Account – Alchemy LMS");
            driver.quit();
        }else{
            System.out.println("The title does not match");
        }

    }
}

File: Selenium/Project/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from webdriver_manager.firefox import GeckoDriverManager

service = webdriver.FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:

    driver.get("https://alchemy.hguy.co/lms")
    driver.find_element(By.XPATH, "//a[text()='My Account']").click()
    title = driver.title

    try:
        assert title == "My Account – Alchemy LMS"
        print("The title of the page is My Account – Alchemy LMS")
    except:
        print("The title does not match")
